### PR TITLE
Minor adjusts to fix the Refresh Data button on watchOS

### PR DESCRIPTION
**Pull request by unnamedd**
Hi @julianschiavo,

first of all, congratulations to you and your project, you are doing such an amazing job sharing these numbers out there to help people understand how hard is the situation. 

Running your project on my iPhone and Watch, I saw a small thing and I'm here to send you a small contribution, but I'm sure it will help. 

I made a small tweak in the ListButton to fix the presentation on the watchOS.

From this:
Simulator Screen Shot - Apple Watch Series 5 - 44mm - 2020-03-16 at 19 53 57

To this:
Simulator Screen Shot - Apple Watch Series 5 - 44mm - 2020-03-16 at 19 53 07

I hope this change would help!

### CODE REVIEW

File: Shared/RegionList.swift [hunk 1]
```diff
@@ -67,8 +67,7 @@ struct RegionList: View {
 }
 
 struct CountryList_Previews: PreviewProvider {
-    @State static var isRefreshAlertPresented = false
```
Comment (julianschiavo) [inline Shared/RegionList.swift]: Please revert these changes to avoid having unrelated changes in the PR

File: Shared/Reusable/ListButton.swift [hunk 2]
```diff
@@ -12,18 +12,29 @@ import SwiftUI
 struct ListButton: View {
     var image: Image
     var text: String
-    var action: () -> Void
+    var action: () -> Void = {}
```
Comment (julianschiavo) [inline Shared/Reusable/ListButton.swift]: Same here, unnecessary

File: Shared/Reusable/ListButton.swift [hunk 3]
```diff
@@ -12,18 +12,29 @@ import SwiftUI
...
     var body: some View {
         Button(action: action) {
             HStack(spacing: 10) {
-                #if os(iOS)
-                HStack { image.font(.system(size: 20)) }
-                    .frame(width: 30)
+                image.font(.system(size: 20))
+                
                 Text(text)
-                #endif
```
Comment (julianschiavo) [inline Shared/Reusable/ListButton.swift:24]: Here can we leave the HStack layout previously used as it is purposefully used to align the images on iOS?

File: Shared/Reusable/ListButton.swift [hunk 4]
```diff
@@ -12,18 +12,29 @@ import SwiftUI
...
+struct ListButton_Previews: PreviewProvider {
+    static var previews: some View {
+        ListButton(
+            image: Images.refresh,
+            text: "Refresh Data"
+        )
+    }
+}
+
+#endif
```
Comment (julianschiavo) [inline Shared/Reusable/ListButton.swift]: Please revert these changes too

**Review by julianschiavo — CHANGES_REQUESTED** [review]
Hey, thanks for the PR & contribution! Could you remove the unrelated change to the previews and keep the previous ListButton code, just removing the conditional `#if os(iOS)`? Should be good to go other than that!

File: Shared/RegionList.swift (hunk 1 above)
Comment (unnamedd) [reply Shared/RegionList.swift]: Okay 👍

File: Shared/Reusable/ListButton.swift (hunk 2 above)
Comment (unnamedd) [reply Shared/Reusable/ListButton.swift]: Okay 👍

File: Shared/Reusable/ListButton.swift (hunk 3 above)
Comment (unnamedd) [reply Shared/Reusable/ListButton.swift:24]: Okay 👍

File: Shared/Reusable/ListButton.swift (hunk 4 above)
Comment (unnamedd) [reply Shared/Reusable/ListButton.swift]: I understood your previous points, but do you mind to explain why revert this preview here? Honestly, I don’t see any reason to not have a preview for some view when we are using SwiftUI. Would you mind share your points?

Comment (julianschiavo) [reply Shared/Reusable/ListButton.swift]: Okay, I agree that we can keep the change but could you remove it here, I’ll merge this PR then you can create another PR with these changes? Just to keep things organized :)

Comment (unnamedd) [reply Shared/Reusable/ListButton.swift]: If you wish, here we go! ;)

**Review by julianschiavo — APPROVED** [review]
Good to go, merged! Thanks for the PR and feel free to PR the preview too so I can get that in :)
Thanks again!
